index.d.ts

```diff
@@ -50,10 +50,10 @@ declare module 'react-intersection-observer' {
     triggerOnce?: boolean
 
     /** Call this function whenever the in view state changes */
-    onChange?(inView: boolean | React.FormEvent<HTMLElement>): void
+    onChange?: (inView: boolean) => void
 
     /** Get a reference to the the inner DOM node */
-    innerRef?(element?: HTMLElement): void
+    innerRef?: (element?: HTMLElement) => void
```

Comment:
Ditto for the syntax.
Typing `innerRef` seems to have loads of variations (https://github.com/search?l=TypeScript&p=1&q=innerRef&type=Code). I have no idea what a definitive one would be. I guess there is an option to make it `element: any`, but I am sure it will surface if someone has a use case for it.